tests/.pylintrc

```diff
@@ -44,6 +44,7 @@ disable=
   missing-class-docstring,
   missing-function-docstring,
   no-self-use
+  bad-continuation
```

Comment:
I had odd bad-continuation alerts on the test file so added this for the moment 🤔